### ISSUE TITLE
OneDrive/SharePoint File Picker not working for DocumentSets

### ISSUE THREAD

**Issue by nicolasiscoding** [OneDrive/onedrive-api-docs#1743]
Hi all,

Figured I should cross post here for extra visibility since this would apply to the API.

I am in the process of onboarding a client using SharePoint and everything is working with the exception of DocumentSets. In my local tenant on O365, I am unable to reproduce this issue, but in the client's tenant, DocSets are being treated as "files" and not folders when clicking on them in the FilePicker api.

Below is some triage I did in the client's tenants, and they made a comment about a setting they may have changed.

Hopping for any guidance or assistance on this one

https://github.com/OneDrive/samples/issues/57

**Comment by nicolasiscoding** [comment]
After reproing this without the sdk itself (which appears to be a link builder at the end of the day), we opened a ticket with Microsoft. The ticket has still not been triaged and for the time being we wrote our own (less featured) file picker using the graph API.

Hope to use this library again soon but for the time being, we are unable to use this with specific clients due to bugs on the file picker side.
